modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -31,3 +31,46 @@ You can install the AWS Load Balancer Operator from the OperatorHub by using the
 .Verification
 
 * Verify that the AWS Load Balancer Operator shows the *Status* as *Succeeded* on the Installed Operators dashboard.
+
+.Installing through the command line
+
+. Create aws-load-balancer-operator namespace: oc new-project aws-load-balancer-operator and the created OperatorGroup and Subscription:
+
+[YAML]
+====
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+name: aws-load-balancer-operator
+namespace: aws-load-balancer-operator
+spec:
+upgradeStrategy: Default
+EOF
+====
+
+[YAML]
+====
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+name: aws-load-balancer-operator
+namespace: aws-load-balancer-operator
+spec:
+channel: stable-v1.1
+installPlanApproval: Automatic
+name: aws-load-balancer-operator
+source: redhat-operators
+sourceNamespace: openshift-marketplace
+EOF
+====
+
+.Verification
+
+Check the status of an install plan. The status of an install plan must be `Complete`:
+
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator get ip <install_plan_name> --template='{{.status.phase}}{{"\n"}}'
+----
```

Comment:
These lines for the steps would be:
+ 
The status must be `complete`
+
. Example output
[source,terminal]
----
complete
----

Reply:
ty